src/app/post-login-banner/drop-banner/drop-banner.component.pug

```diff
@@ -1,7 +1,7 @@
 div.drop-tray(@trayAnimation)
   .container
     .wrapper
-      img.logo(src="/assets/img/artstor-jstor-logo.svg", alt="Artstor JSTOR Logo")
```

Comment:
can this file be deleted now?

Reply:
Yep. Nice catch. Thanks for pointing it out.